src/app/places/places.component.html

```diff
@@ -1,20 +1,22 @@
 <div class="container">
   <ul class="places">
-    <li data-test="list-places" *ngFor="let place of PlacesFromService">
-      <div class="img">
-        <img class="images" src="../../assets/images/{{ place.Id }}.jpg" />
-      </div>
-      <div class="text">
-        <h3>{{ place.Name }}</h3>
-        {{ place.Description }}
-        <h5>
-          {{ place.Address }}, Tel: {{ place.Phone }}, Link do strony:
-          <a href="{{ place.Link }}" target="_blank">{{ place.Link }}</a
-          ><br />Tagi: {{ place.Tag }}<br />
-          Max ilość osób: {{ place.Max_persons }}
-        </h5>
-        <br />
-      </div>
-    </li>
+  <li data-test="list-places" *ngFor="let place of PlacesFromService | paginate: { itemsPerPage: 5, currentPage: p }">
+  <div class="img">
+  <img class="images" src="../../assets/images/{{ place.Id }}.jpg" />
+  </div>
+  <div class="text">
+  <h3>{{ place.Name }}</h3>
+  {{ place.Description }}
+  <h5>
+  {{ place.Address }}, Tel: {{ place.Phone }}, Link do strony:
+  <a href="{{ place.Link }}" target="_blank">{{ place.Link }}</a
+  ><br />Tagi: {{ place.Tag }}<br />
+  Max ilość osób: {{ place.Max_persons }}
+  </h5>
+  <br />
+  </div>
+  </li>
   </ul>
-</div>
+  <pagination-controls (pageChange)="p = $event"></pagination-controls>
+  </div>
+  
```

Comment:
It seems that prettier isn't working correctly. Could You please report a bug in your repo?

Reply:
Done